playbooks/openshift-node/private/image_prep.yml

```diff
@@ -6,6 +6,13 @@
     skip_sanity_checks: True
     skip_validate_hostnames: True
 
+- name: determine version
+  import_playbook: ../../init/version.yml
```

Comment:
I don't mind this approach but this uncovers some inadequacies regarding the `prerequisites.yml` above.  

We probably need to remove version from the prerequisites.yml and move to the approach you have here in other places.

Thoughts?

Reply:
prerequisites.yml skips version on purpose.  That's because we need a container runtime to be installed to query for exact image tags.

prerequisites.yml ensures that registries and authentication is in place so we can actually run openshift_version.  I'm a proponent of doing away with openshift_version entirely, but that's not where we're at today.

Reply:
I won't block on this but it seems counterintuitive to call prerequisities.yml and skip openshift_version and turn right around and call openshift_version.  Could we just pass the variables to prerequisites.yml?

Reply:
@kwoodson prerequisites.yml is a prerequisite for openshift_version, that's why prerequisites.yml skips openshift_version.

We can't run openshift_version without container runtime being installed and configured.

Reply:
@michaelgugino @mtnbikenc 

We discussed this in a bjn.  We will think about modularizing the `init/main.yml` path so that it is clear and concise.

Thanks.